src/App.jsx

```diff
@@ -2,10 +2,12 @@ import { initialColors } from "./lib/colors";
 import Color from "./Components/Color/Color.jsx";
 import "./App.css";
 import ColorForm from "./Components/Form/ColorForm.jsx";
-import { useState } from "react";
+import useLocalStorageState from "use-local-storage-state";
 
 export default function App() {
-  const [colors, setColors] = useState(initialColors);
+  const [colors, setColors] = useLocalStorageState("colors", {
+    defaultValue: [initialColors],
```

Comment:
looks good. Does it make a difference when you wrap the initialColors inside []? I don´t have this but I will look into it.

Reply:
assuming initialColors is already an array. Wrapping it in additional brackets would turn it into an array of arrays, which would likely cause issues when working with the color data

Reply:
Good to know, thanks a lot! I have fixed it.